### DISCUSSION TITLE
what is the purpose of having auto cmd and auto groups on legendary?

### DISCUSSION THREAD

**Discussion by danielo515** [mrjones2014/legendary.nvim#124] [Q&A] [answered]
Sorry for the silly question, but why will you want to have a ser of commands that are supposed to execute automatically within a command pallette?

**Comment by mrjones2014** [comment, marked as answer]
Legendary is more than just the command palette, in fact that was something I realized I could implement after I started building it.

legendary is primarily a better and easier way to do keymaps, commands, and autocmds in pure Lua. 

This is why autocmds don’t show in the finder unless you’ve added a description by default.
